MovieApp/MovieApp/App/Features/MovieSearch/Components/MovieCell.swift

```diff
@@ -22,7 +23,11 @@ class MovieCell: UITableViewCell {
     }
 
     func populateCell(with movie: Movie) {
-        movieImageView.image = UIImage(named: movie.image)
+        let movieUrl = movie.imageURL
+        
+        guard let URL = URL(string: "https://image.tmdb.org/t/p/original\(movieUrl)") else { return }
```

Comment:
Name of variables is lowercased. 
Movie Cell should not know the path for his url.
You can map the models in presenter for now